src/qt/paymentserver.cpp

```diff
@@ -408,6 +297,10 @@ void PaymentServer::handleURIOrFile(const QString& s)
         QUrlQuery uri((QUrl(s)));
         if (uri.hasQueryItem("r")) // payment request URI
         {
+#ifdef ENABLE_BIP70
+            Q_EMIT message(tr("URI handling"),
+                tr("You are using a BIP70 URL which will be unsupported in the future."),
+                CClientUIInterface::ICON_WARNING);
```

Comment:
14451: This should be reverted / commented out. We have no intention of removing BIP70 support.

Reply:
Let's comment it out, should hopefully be better for future backports and cherry-picks to produce less conflicts.